modules/videoio/cmake/detect_ximea.cmake

```diff
@@ -6,10 +6,12 @@ if(NOT HAVE_XIMEA)
     endif()
   endif()
   if(X86_64)
-    set(lib_dir "x64")
+    set(lib_dir1 "API/x64")
+    set(lib_dir2 "API/64bit")
     set(lib_suffix "64")
   else()
-    set(lib_dir "x86")
+    set(lib_dir1 "API/x86")
```

Comment:
I think `if(X86)` check should be added. OpenCV supports ARM, PowerPC, etc.

Reply:
Perhaps. Do you have the opportunity to check the ximea folder tree on an ARM board for example? I have only been able to test it on X86 boards using Windows or Linux...
However i have not changed the way it was previously done and if the architecture is not x86, the folder will not be found and it should not be a problem.

Reply:
Also I propose to use CMake list instead of lib_dirXXX. It's native interface for `FIND_LIBRARY` and easy to extend in future.